proxy/json.go

```diff
@@ -45,6 +46,25 @@ func (j jsonObject) String(key string) (string, error) {
 	return result, nil
 }
 
+func (j jsonObject) Int(key string) (int, error) {
```

Comment:
The underlying map value type for JSON number fields is `json.Number`, not `string` - if you try to use the `String()` method above on it you will get an `UnmarhsalWrongTypeError`